tool_path_planner/src/plane_slicer_raster_generator.cpp

```diff
@@ -681,7 +714,15 @@ boost::optional<ToolPaths> PlaneSlicerRasterGenerator::generate()
   }
 
   // converting to poses msg now
-  rasters = convertToPoses(rasters_data_vec);
+  if (config_.generate_extra_rasters)
```

Comment:
If `generate_extra_rasters` was an integer you could run a subsequent for loop that generates the requested number of "extra rasters"

Reply:
TODO: make extra_rasters an unsigned integer, defaulted to zero, when non-zero, generate as many as requested